src/erfam.h

```diff
@@ -41,6 +41,12 @@ typedef struct {
    double pv[2][3];   /* barycentric PV of the body (au, au/day) */
 } eraLDBODY;
 
+/* Leap second definition */
+typedef struct {
+   int iyear, month;
+   double delat;
+} eraLEAPSECOND;
+
```

Comment:
The new public leap second structure, that comes along with `eraUpdateLeapSeconds()`.